docs/architecture/tokens.md

```diff
@@ -5,32 +5,35 @@ last_update:
   date: 28 November 2023
 ---
 
-In a blockchain ecosystem, a digital asset that can be transferred between accounts is called a _token_.
-Tezos supports many types of tokens, including:
+Any blockchain ecosystem relies on a native token that can be transferred between accounts and in which transaction fees are paid in.
```

Comment:
The info about how blockchain ecosystems work is more suited for opentezos. Here we should keep the info to the behavior of Tezos itself for developers.

Reply:
OK, I re-introduced the original first phrase, and recentered this one on Tezos.